lib/alert_if_404_url_present.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'config'
+
+masterlist = ARGV[0]
+
+known_good = []
+CSV.foreach(masterlist, 'r') do |row| # assume each line sorted by basepath
+  known_good << row[0]
+end
+
+$stdin.each_line do |line|
```

Comment:
A comment with an example value for `line` here would make it clear what we can expect things like `fragment[-2]` to be.